doc/migrations/v0.46-v1.0.0.md

```diff
@@ -365,6 +366,52 @@ const node = await createLibp2p({
 })
 ```
 
+## Keeping the same PeerId after a restart
```

Comment:
```suggestion
### Keeping the same PeerId after a restart
```

I would suggest moving this section directly below the `###Keychain` section as a subsection of `##Misc`

Reply:
My opinion is not terribly strong but `Misc` is under `Extracted modules` so I think it might mess up the hierarchy a little.

The keychain module was extracted, but I feel this is important enough to have it's own section?

Reply:
If the keychain module is now an extracted module and this relates to the keychain doesn't that suggest it should be associated with the Keychain subsection `Misc` section?

Reply:
It felt a bit buried down there.